src/config/routes/2.1.json

```diff
@@ -174,6 +174,20 @@
     },
 
     {
+        "path": "/categories$",
```

Comment:
Most of the other routes have `/?` at the end to cope with a trailing slash or not; please add this in.